modules/scripts/startup-script/files/install_ansible.sh

```diff
@@ -39,7 +39,8 @@ apt_wait() {
 install_python_deps() {
 	if [ -f /etc/debian_version ] || grep -qi ubuntu /etc/lsb-release 2>/dev/null ||
 		grep -qi ubuntu /etc/os-release 2>/dev/null; then
-		apt-get install -y python3-distutils
+		apt-get update
```

Comment:
I cc'ed you on a task named "address release info changes for apt-get update". Please take a look and address here if needed.

Reply:
Ignore. I see this is proposed in separate pr.